Userland/Libraries/LibJS/Runtime/TypedArray.cpp

```diff
@@ -257,15 +266,15 @@ void TypedArrayBase::visit_edges(Visitor& visitor)
     }                                                                                                                                  \
                                                                                                                                        \
     /* 23.2.5.1 TypedArray ( ...args ), https://tc39.es/ecma262/#sec-typedarray */                                                     \
-    Value ConstructorName::construct(FunctionObject&)                                                                                  \
+    Value ConstructorName::construct(FunctionObject& new_target)                                                                       \
     {                                                                                                                                  \
         auto& vm = this->vm();                                                                                                         \
         if (vm.argument_count() == 0)                                                                                                  \
-            return ClassName::create(global_object(), 0);                                                                              \
+            return ClassName::create(global_object(), 0, new_target);                                                                  \
                                                                                                                                        \
         auto first_argument = vm.argument(0);                                                                                          \
         if (first_argument.is_object()) {                                                                                              \
-            auto* typed_array = ClassName::create(global_object(), 0);                                                                 \
+            auto* typed_array = ClassName::create(global_object(), 0, new_target);                                                     \
```

Comment:
This probably needs an exception check now

Reply:
yep, fixed